packages/checkout/slot/index.js

```diff
@@ -2,13 +2,13 @@
  * External dependencies
  */
 import deprecated from '@wordpress/deprecated';
+import { CURRENT_USER_IS_ADMIN } from '@woocommerce/settings';
+import { Children, cloneElement } from '@wordpress/element';
 import {
 	createSlotFill as baseCreateSlotFill,
 	__experimentalUseSlot,
-	useSlot as __useSlot,
+	useSlot as __useSlot, //eslint-disable-line
```

Comment:
Without the comment this throws a warning, I think that's ok not to disable, it won't break anything.

`  10:2  warning  useSlot not found in 'wordpress-components'  import/named`